python/examples/mysql.py

```diff
@@ -0,0 +1,70 @@
+import pymysql
+from pshconfig import Config
+
+
+def test_output():
+
+    # Create a new Config object to ease reading the Platform.sh environment variables.
+    # You can alternatively use os.environ yourself.
+    config = Config()
+
+    # The 'database' relationship is generally the name of primary SQL database of an application.
+    # That's not required, but much of our default automation code assumes it.'
+    credentials = config.credentials('database')
+
+    try:
+
+        # Connect to the database using PDO. If using some other abstraction layer you would inject the values
+        # from `database` into whatever your abstraction layer asks for.
+        conn = pymysql.connect(host=credentials['host'],
+                               port=credentials['port'],
+                               database=credentials['path'],
+                               user=credentials['username'],
+                               password=credentials['password'])
+
+        sql = "CREATE TABLE People (" \
+              "id SERIAL PRIMARY KEY," \
+              "name VARCHAR(30) NOT NULL," \
+              "city VARCHAR(30) NOT NULL" \
+              ")"
```

Comment:
Use triple-quoted strings here instead of \.  It's easier to read and easier to maintain.

Reply:
Same thing elsewhere in these examples; I won't call out each one to avoid spam.